google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/models/Query.java

```diff
@@ -59,8 +61,64 @@ public static Query create(String tableId) {
     return new Query(tableId);
   }
 
-  private Query(String tableId) {
+  /**
+   * Constructs a new Query object with the specified authorized view id, which permits access to an
+   * explicit subset of the specified table. The authorized view id and table id will be combined
+   * with the instance name specified in the {@link
+   * com.google.cloud.bigtable.data.v2.BigtableDataSettings}.
+   *
+   * <p>See {@link com.google.cloud.bigtable.admin.v2.models.AuthorizedView} for more details about
+   * an AuthorizedView.
+   */
+  public static Query createForAuthorizedView(String tableId, String authorizedViewId) {
+    return new Query(tableId, authorizedViewId);
+  }
+
+  @InternalApi("For internal use only")
```

Comment:
same question here, where is this used? could it be pacakge private?

Reply:
Let's close this one and discuss in the comment below.